reconcile/openshift_base.py

```diff
@@ -252,6 +253,24 @@ def apply(dry_run, oc_map, cluster, namespace, resource_type, resource,
             oc.delete(namespace=namespace, kind=resource_type,
                       name=resource.name)
             oc.apply(namespace=namespace, resource=annotated)
+        except StatefulSetUpdateForbidden:
+            if resource_type not in ['StatefulSet']:
+                raise
+
+            logging.info(['delete_and_apply', cluster, namespace,
+                          resource_type, resource.name])
+            owned_pods = oc.get_owned_pods(namespace, resource)
+            oc.delete(namespace=namespace, kind=resource_type,
+                      name=resource.name, cascade=False)
+            oc.apply(namespace=namespace, resource=annotated)
+            logging.info(['recycle_pods', cluster, namespace,
```

Comment:
For the same above reasons
```suggestion
            logging.info(['recycle_sts_pods', cluster, namespace,
```